.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: crystal
```

Comment:
@drujensen I would not know how to do this. Let me know when you have some time I would like if you can walk me thru this setup.